knowledge/faq.rst

```diff
@@ -122,7 +122,7 @@ Conan will not normalize or change in any way the source files, it is not its re
 
 .. code-block:: ini
 
-* text eol=lf
```

Comment:
Why exactly? Isn't the ``*`` pattern working correctly, or is it just to simplify?
Thanks!

Reply:
this is how it looks at the moment:

![Screenshot 2025-05-14 at 14 36 48](https://github.com/user-attachments/assets/1eceb9e5-767b-4cdb-b5c7-368cf2c8b053)

Reply:
oh, I thought that `*` was a mistakenly used list element. It should be a wildcard pattern, right?